yaksh/models.py

```diff
@@ -68,43 +68,6 @@
                 ('completed', 'Completed'),
               )
 
-instructions_data = dedent("""\
```

Comment:
This has been moved to forms.py in a previous PR. The default instructions is initialized from forms and not models. Hence it is no longer required in models.